packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -170,6 +173,7 @@ export const Popup: React.FC<PopupProps> &
     unstable_disableTether,
     unstable_pinned,
     autoSize,
+    closeOnScroll,
```

Comment:
let's have a test in cypress for this, it should be pretty straightforward to trigger a scroll event on the window

Reply:
done :)